src/toil/test/provisioners/clusterTest.py

```diff
@@ -40,7 +41,12 @@ def __init__(self, methodName: str) -> None:
         # We need a boto2 connection to EC2 to check on the cluster.
         # Since we are protected by needs_aws_ec2 we can import from boto.
         import boto.ec2
-        self.boto2_ec2 = boto.ec2.connect_to_region(zone_to_region(self.zone))
+        self.region = zone_to_region(self.zone)
+        self.boto2_ec2 = boto.ec2.connect_to_region(self.region)
```

Comment:
Shouldn't this come off the AWSConnectionManager()?

Reply:
This should be fixed in my next PR, I mainly kept the boto2 part here to avoid spending too much time on porting over tests